examples/inputs/turbine_definitions/iea_10MW.yaml

```diff
@@ -0,0 +1,164 @@
+turbine_type: 'iea_10MW'
+generator_efficiency: 1.0
+hub_height: 119.0
```

Comment:
Could potentially move this to the main input file.

Reply:
See above comment.